packages/renderer/src/lib/compose/ComposeDetails.spec.ts

```diff
@@ -54,7 +54,8 @@ beforeAll(() => {
       func();
     },
   };
-  (window as any).getConfigurationValue = vi.fn();
+  (window as any).getConfigurationValue = vi.fn().mockResolvedValue(undefined);
+  (window as any).getConfigurationProperties = vi.fn().mockResolvedValue(undefined);
```

Comment:
issue: no undefined, only {} is possible

Reply:
does not really matters for test, could be a sign of wrong exception handling, but it out of scope of the PR.